cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
+      DomainName: "test"
+      ElasticsearchClusterConfig:
+        DedicatedMasterEnabled: true
+        InstanceCount: 2
+        ZoneAwarenessEnabled: true
+        InstanceType: "m3.medium.elasticsearch"
+        DedicatedMasterType: "m3.medium.elasticsearch"
+        DedicatedMasterCount: 3
+      EncryptionAtRestOptions:
```

Comment:
Coverity found issue: Disk-level encryption is disabled - CWE-313, Medium Severity

****

**How to fix:** Enable encryption for data at rest by explicitly setting the `Properties.EncryptionAtRestOptions.Enabled` value to `true`.

<details>
<summary>Click to expand data flow...</summary>

**From cft/deploy.yml:**
```
   10         InstanceType: "m3.medium.elasticsearch"
   11         DedicatedMasterType: "m3.medium.elasticsearch"
   12         DedicatedMasterCount: 3
1. Sigma main event: The Amazon ElasticSearch Domain does not have encryption enabled for data at rest.

2. remediation: Enable encryption for data at rest by explicitly setting the `Properties.EncryptionAtRestOptions.Enabled` value to `true`.

   13       EncryptionAtRestOptions:
   14         Enabled: false
   15       EBSOptions:
```
</details><!-- Coverity aab1fd3acb4880010d15971b519031b0 -->